packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -2637,6 +2643,9 @@ class InputDecoration {
   /// See [Icon], [ImageIcon].
   final Widget icon;
 
+  /// If specified, the icon color used for the default [icon] color
```

Comment:
this documentation is highly unhelpful. I can tell it's the icon color, the name is "icon color". :-)

This should say things like what the default is, how it's applied, whether it can be null, when it takes effect and when it gets overridden, etc.

Reply:
see https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc

Reply:
Thank you for the link, I improved the documentation and hope that it fits now better to the expectations.